tests/test_metrics.py

```diff
@@ -884,3 +884,31 @@ def test_get_ignored_features(self):
         ignored_features = collection.get_ignored_features()
         expected_ignored_features = ("indegree_1", "outdegree_1")
         self.assertTrue(ignored_features == expected_ignored_features)
+
+
+class TestSumDistancesConnectedNeurons(unittest.TestCase):
+    def test_sum_distances(self):
+        positions = pd.DataFrame({"x_pos": [0, 0, 4], "y_pos": [0, 3, 0], "z_pos": [2, 2, 2]})
+        W = np.array([[0, 1, 0],
+                      [1, 0, 1],
+                      [1, 0, 0]])
+
+        # Dataframe with multiple columns
+        metric_1 = SumDistancesConnectedNeurons(positions)
+        expected_res_1 = 3 + 3 + 5 + 4
+        self.assertTrue(metric_1.calculate(W) == expected_res_1)
+
+        # 2D numpy array
+        metric_2 = SumDistancesConnectedNeurons(positions.to_numpy())
+        expected_res_2 = 3 + 3 + 5 + 4
+        self.assertTrue(metric_2.calculate(W) == expected_res_2)
+
+        # Series
+        metric_3 = SumDistancesConnectedNeurons(positions.x_pos)
+        expected_res_3 = 4 + 4
+        self.assertTrue(metric_3.calculate(W) == expected_res_3)
+
+        # 1D numpy array
+        metric_4 = SumDistancesConnectedNeurons(positions.z_pos.to_numpy())
```

Comment:
why is this 0?

Reply:
In this example all neurons are on the same z-position (2), so the distance on z-axis is 0

Reply:
You're right, I don't know what I thought...